src/controllers/confirmation.controller.ts

```diff
@@ -27,9 +29,19 @@ const route = async (req: Request, res: Response, next: NextFunction): Promise<v
   }
   // TODO LFA-1169 Clarify what the ext deadline period will be for accounts and cs.
 
-  // TODO LFA-TBC call promise-to-file api
-
   const isStillRequired: boolean = getPromiseToFileSessionValue(req.chSession, IS_STILL_REQUIRED);
+
+  const token: string = req.chSession.accessToken() as string;
+  try {
+    if (token) {
+      // TODO  LFA-TBC Add isSubmitted flag to prevent this being sent twice
```

Comment:
We should create the JIRA task at this point and reference it here, rather than use TBC. I can't actually see anything in the story we're doing about preventing duplicate submissions so does this need clarifying by an analyst?

Reply:
Created ticket LFA-1406 and updated TODO comment.